src/sbl_filing_api/services/request_handler.py

```diff
@@ -0,0 +1,24 @@
+import logging
+
+import httpx
+from sbl_filing_api.config import settings
+
+logger = logging.getLogger(__name__)
+
+
+def send_confirmation_email(
+    user_full_name: str, user_email: str, contact_info_email: str, confirmation_id: str, timestamp: int
+):
+    confirmation_request = {
+        "confirmation_id": confirmation_id,
+        "signer_email": user_email,
+        "signer_name": user_full_name,
+        "contact_email": contact_info_email,
+        "timestamp": timestamp,
+    }
+    try:
+        res = httpx.post(settings.mail_api_url, json=confirmation_request)
+        if res.status_code != 200:
+            logger.error(res.text)
```

Comment:
I think we'd want to log this result no matter what?  It'll be the log record of what email was sent.
So maybe a log.info or log.debug of res.text prior to the status_code check?